INSTALL

```diff
@@ -168,6 +173,26 @@ When updating MORSE to a more recent version, you'll simply have to do::
     $ cd build
     $ make install
 
+Windows
+-------
+
+Download the latest version of the source code. It is stored in a git
+repository::
+
+  $ git clone https://github.com/morse-simulator/morse.git
+
+The MORSE_BLENDER environment variable should be set the the location and filename
```

Comment:
There is written two times "the" between "be set" and "location and filename".

Reply:
Fixed!